rpc/rpc.go

```diff
@@ -399,7 +399,13 @@ func (c *Conn) releaseAnswers(answers map[answerID]*answer) {
 
 func (c *Conn) releaseQuestions(questions []*question) {
 	for _, q := range questions {
-		q.Reject(ExcClosed)
+		canceled := q != nil && q.flags&finished != 0
```

Comment:
Minor thing that totally shouldn't be fixed now:  maybe add methods to a `flags` type, so that we can do `q.flags.Finished()`?

Reply:
That seems reasonable. Or maybe at least a flags.Contains(finished), which would be less boilerplate to implement.